stdlib/experimental/chain.go

```diff
@@ -46,10 +46,10 @@ func chainCall(ctx context.Context, args values.Object) (values.Value, error) {
 		return nil, errors.Wrap(err, codes.Inherit, "error in table object compilation")
 	}
 
-	if !execdeps.HaveExecutionDependencies(ctx) {
+	if !execute.HaveExecutionDependencies(ctx) {
 		return nil, errors.New(codes.Internal, "no execution context for chain to use")
 	}
-	deps := execdeps.GetExecutionDependencies(ctx)
+	deps := execute.GetExecutionDependencies(ctx)
```

Comment:
Does `chain()` also need to inject the profiler into the context like `tableFind`?

Reply:
With the latest commit, the need to inject the profiler into the context in `tableFind` and `chain`, etc goes away.